### DISCUSSION TITLE
can we change the logo templates?

### DISCUSSION THREAD

**Discussion by smarthashmi** [Codeinwp/logo-maker#75] [Q&A]
we are not able to change the logo template's

**Comment by Soare-Robert-Daniel** [comment]
What do you mean exactly? Replacing the current logo SVG with other ones?

**Reply by smarthashmi** [reply]
yes i have svg and want to replace it with it

**Reply by Soare-Robert-Daniel** [reply]
You need to look at this folder https://github.com/Codeinwp/logo-maker/tree/main/src/assets/logos

They are coded as TSX files. When the client makes a request, he gets all the logos at once. Also, it is easy to cache the response.

**Reply by smarthashmi** [reply]
so i cant use my templates is TSX ? what if we give privilege to admin he can upload the svg from admin side?

**Reply by Soare-Robert-Daniel** [reply]
You can not do it. The SVG is hardcoded into the code base. It was not made as _extensible_ via outside actions.

You have to implement this functionality.

**Reply by smarthashmi** [reply]
i want to do it can you please guide me on it....

**Reply by Soare-Robert-Daniel** [reply]
Unfortunately no. But you can use tools like ChatGPT to guide you. Since the project is older, it was probably used as a training set.

Let the AI revolution shine upon us.
